composer.json

```diff
@@ -1,46 +1,62 @@
 {
-    "name":         "behat/mink-selenium2-driver",
-    "description":  "Selenium2 (WebDriver) driver for Mink framework",
-    "keywords":     ["selenium", "webdriver", "javascript", "ajax", "testing", "browser"],
-    "homepage":     "http://mink.behat.org/",
-    "type":         "mink-driver",
-    "license":      "MIT",
-
+    "name": "behat/mink-selenium2-driver",
+    "description": "Selenium2 (WebDriver) driver for Mink framework",
+    "keywords": [
+        "selenium",
+        "webdriver",
+        "javascript",
+        "ajax",
+        "testing",
+        "browser"
+    ],
+    "homepage": "http://mink.behat.org/",
+    "type": "mink-driver",
+    "license": "MIT",
     "authors": [
         {
-            "name":      "Pete Otaqui",
-            "email":     "pete@otaqui.com",
-            "homepage":  "https://github.com/pete-otaqui"
+            "name": "Pete Otaqui",
+            "email": "pete@otaqui.com",
+            "homepage": "https://github.com/pete-otaqui"
+        },
+        {
+            "name": "Konstantin Kudryashov",
+            "email": "ever.zet@gmail.com",
+            "homepage": "http://everzet.com"
+        }
+    ],
+    "repositories": [
```

Comment:
Surely this change isn't meant to be merged.

Reply:
right, this PR is currently WIP.

Why it's needed:
- facebook/php-webdriver has an issue with detecting related <form/> element (already submitted https://github.com/facebook/php-webdriver/pull/621)
- mink/driver-testsuite is required because of https://github.com/minkphp/driver-testsuite/pull/25 (facebook/php-webdriver is a bit more w3c complaint and w3c specifications does not have keyup/keydown specifications)